deploy/manifests/prod/us-east-2/tenant/storetheindex/dhstore/pvc-gp3.yaml

```diff
@@ -8,8 +8,4 @@ spec:
   resources:
     requests:
       storage: 16Ti
-  dataSource:
-    name: dhstore-20230427
-    kind: VolumeSnapshot
-    apiGroup: snapshot.storage.k8s.io
-  storageClassName: gp3-iops5k-t300
```

Comment:
What's the rationale for removing this? I am not sure if pvc will be modifiable this way. If unrelated to pr i recommend restoring these changes.

Reply:
I removed it because I thought we could stop using the snapshot here, and that would allow us to delete it later. I will put it back to make sure that it does not cause a problem.